desktop/extensions-sdk/build/set-up/minimal-backend-extension.md

```diff
@@ -1,18 +1,22 @@
 ---
-title: Set up a minimal backend extension 
+title: Set up a minimal backend extension
 description: Minimal backend extension tutorial
 keywords: Docker, extensions, sdk, build
 redirect_from:
-- /desktop/extensions-sdk/tutorials/minimal-backend-extension/
+  - /desktop/extensions-sdk/tutorials/minimal-backend-extension/
 ---
 
 To start creating your extension, you first need a directory with files which range from the extension’s source code to the required extension-specific files. This page provides information on how to set up a simple Docker Extension that runs CLI commands in the backend.
 
 For extensions with a backend service running REST services over sockets or named pipes, see the `vm-ui extension` [sample](https://github.com/docker/extensions-sdk/tree/main/samples).
 
->Note
+> Note
 >
->Before you start, make sure you have installed the latest version of [Docker Desktop](../../../release-notes.md).
+> Before you start, make sure you have installed the latest version of [Docker Desktop](../../../release-notes.md).
+
+> Note
+>
+> If you want to start a codebase for your new extension, our [Quickstart guide](../../quickstart.md) and `docker extension init <my-extension>` will provide a better base for your extension, more up-to-date and related to your install of Docker Desktop.
```

Comment:
```suggestion
> If you want to start a codebase for your new extension, our [Quickstart guide](../../quickstart.md) and `docker extension init <my-extension>` provides a better base for your extension as it is more up-to-date and related to your install of Docker Desktop.
```

Reply:
Sorry @gtardif, I should've added that this needs to be amended on each page where it is repeated